src/test/java/com/wikia/webdriver/common/core/configuration/Configuration.java

```diff
@@ -157,6 +157,23 @@ public static String getCountryCode() {
     return getProp("countryCode");
   }
 
+  /**
+   * @return null if window is supposed to be maximised, Dimension if any other size is demanded
+   */
+  public static Dimension getBrowserSize() {
+    String size = getProp("browserSize");
```

Comment:
it's the same as emelent.getSize, also returns Dimension object. I'm leaving it as it is.